### PR TITLE
Add Roxxe detection

### PR DESCRIPTION
**Pull request by TheRogueArchivist**

### CODE REVIEW

File: BinaryObjectScanner/Protection/Roxxe.cs [hunk 1]
```diff
@@ -0,0 +1,79 @@
...
+        /// <inheritdoc/>
+        public string? CheckPortableExecutable(string file, PortableExecutable pex, bool includeDebug)
+        {
+            // Get the sections from the executable, if possible
+            var sections = pex.Model.SectionTable;
+            if (sections == null)
+                return null;
+
+            // Get the CODE section strings, if they exist
+            var strs = pex.GetFirstSectionStrings("CODE");
```
Comment (mnadareski) [inline BinaryObjectScanner/Protection/Roxxe.cs:29]: Look around the rest of the code, but I _think_ this section can be called both `code` and `CODE`.

File: BinaryObjectScanner/Protection/Roxxe.cs [hunk 2]
```diff
@@ -0,0 +1,79 @@
...
+            if (strs != null)
+            {
+                // Found in "Owar.exe" in IA item "game4u-22-cd".
+                if (strs.Any(s => s.Contains("TRCHANGER.INI")))
+                    return "Roxxe";
+            }
+
+            // Get the .rsrc section strings, if they exist
+            strs = pex.GetFirstSectionStrings(".rsrc");
```
Comment (mnadareski) [inline BinaryObjectScanner/Protection/Roxxe.cs:38]: Did you find that these live in any specific resource? If you can lock it down to a resource and not just the resource section, for example, a string table resource, please do that.

File: README.md [hunk 3]
```diff
@@ -94,6 +94,7 @@ Below is a list of protections detected by BinaryObjectScanner. The two columns
 | Rainbow Sentinel | True | True | |
 | Ring PROTECH / ProRing | True | True | Partially unconfirmed² |
 | RipGuard | True | True | Partially unconfirmed² |
+| Roxxe | True  | False | |
```
Comment (mnadareski) [inline README.md:97]: Nitpick: Fix spacing after `True`

File: BinaryObjectScanner/Protection/Roxxe.cs (hunk 2 above)
Comment (TheRogueArchivist) [reply BinaryObjectScanner/Protection/Roxxe.cs:38]: Despite my best efforts, I have not been able to narrow this down yet. It also wasn't in application-defined resources despite that looking promising.

Comment (TheRogueArchivist) [reply BinaryObjectScanner/Protection/Roxxe.cs:38]: My best efforts weren't good enough

**Review by mnadareski — APPROVED** [review]

**Pull request merged**